dataflow/snippets/src/main/java/com/example/dataflow/BigQueryReadFromQuery.java

```diff
@@ -17,14 +17,14 @@
 package com.example.dataflow;
 
 // [START dataflow_bigquery_read_query]
-import com.google.api.services.bigquery.model.TableRow;
+import com.google.common.collect.ImmutableMap;
```

Comment:
This is cool; it's use for the configuration of the BigQuery read query that ensures that this doesn't change for the pipeline execution seems like a good practice.